sierra_adapter/sierra_reader/docker-compose.yml

```diff
@@ -1,5 +1,7 @@
 s3:
-  image: "760097843905.dkr.ecr.eu-west-1.amazonaws.com/scality/s3server:mem-latest"
```

Comment:
See also: https://github.com/wellcomecollection/platform/issues/5547

The scality/s3server image is quite old, and doesn't support the ListObjectsV2 method, which we need!